### PR TITLE
MUMUP-2711 no popups in guest mode

### PR DESCRIPTION
**Pull request by Doug-Reed**

### CODE REVIEW

**Comment by vertein** [comment]
:+1:  - Also, going to do another PR with a guest mode fix for mascot features?

**Comment by Doug-Reed** [comment]
@vertein 

https://github.com/UW-Madison-DoIT/uw-frame/blob/master/uw-frame-components/portal/features/controllers.js

Line 106 in the init function already has a check for guest mode. That should prevent public mascot announcement, shouldn't it? I'll test it.

File: uw-frame-components/portal/features/controllers.js [hunk 1]
```diff
@@ -53,10 +53,10 @@ define(['angular','require'], function(angular, require) {
 
 
      //local functions ---------------------------------------------------------
-     
+
      var getPopups = function(){
        portalFeaturesService.getUnseenPopups().then(function(unSeenPopups) {
-         if(unSeenPopups.length !=0){
+          if(unSeenPopups.length !=0 && !$rootScope.GuestMode===true){
```
Comment (apetro) [inline uw-frame-components/portal/features/controllers.js]: Can `!$rootScope.GuestMode===true` be idiomatically refactored to `!$rootScope.GuestMode`?

Comment (Doug-Reed) [inline uw-frame-components/portal/features/controllers.js]: Who are you calling an idiom?

**Comment by Doug-Reed** [comment]
@apetro : Refactored.

File: uw-frame-components/portal/features/controllers.js [hunk 2]
```diff
@@ -56,7 +56,7 @@ define(['angular','require'], function(angular, require) {
 
      var getPopups = function(){
        portalFeaturesService.getUnseenPopups().then(function(unSeenPopups) {
```
Comment (paulerickson) [inline uw-frame-components/portal/features/controllers.js]: "unseen" is one word, not two!

Comment (vertein) [inline uw-frame-components/portal/features/controllers.js]: I like how @paulerickson swoops in!  I'll take blame, as I did that originally... :(  I will also point out my inconsistency with the getUnseenPopups name and the unSeenPopups variable.

File: uw-frame-components/portal/features/controllers.js [hunk 3]
```diff
@@ -53,10 +53,10 @@ define(['angular','require'], function(angular, require) {
 
 
      //local functions ---------------------------------------------------------
-     
+
      var getPopups = function(){
        portalFeaturesService.getUnseenPopups().then(function(unSeenPopups) {
-         if(unSeenPopups.length !=0){
+          if(unSeenPopups.length !=0 && !$rootScope.GuestMode){
```
Comment (timlevett) [inline uw-frame-components/portal/features/controllers.js]: Should we even call the `portalFeaturesService.getUnseenPopups()` if you are a guest? I would think this check is before this return from a service call.

**Pull request merged**